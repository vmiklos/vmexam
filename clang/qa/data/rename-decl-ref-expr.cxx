class C
{
public:
    static const int aS[];
    static const int* getS() { return aS; }
};

int foo(const int* pS)
{
    return 0;
}
#define FOO(a) foo(a)

int main()
{
    FOO(C::aS);
}
