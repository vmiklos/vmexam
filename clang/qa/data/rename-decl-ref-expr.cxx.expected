class C
{
public:
    static const int m_aS[];
    static const int* getS() { return m_aS; }
};

int foo(const int* pS)
{
    return 0;
}
#define FOO(a) foo(a)

int main()
{
    FOO(C::m_aS);
}
